wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/reader/ReaderScreen.kt

```diff
@@ -155,12 +138,15 @@ fun ReaderScreen(
                 onNavigate
             )
         }
+
```

Comment:
There are a bunch of whitespace and line-wrap reformatting changes in this file. Do we need these? Are these automated from a linter?